src/noted.cpp

```diff
@@ -0,0 +1,82 @@
+#include <argp.h>
+#include <iostream>
```

Comment:
There is already iostream and argp in note.hpp included

Reply:
I don't think argp is included. Removed all unneeded includes.